packages/react/src/patterns/sub-patterns/Card/Card.js

```diff
@@ -89,7 +89,9 @@ function renderFooter(cta, type) {
     cta && (
       <div className={`${prefix}--card__footer`}>
         {type !== 'link' ? (
-          <CTA style="text" {...cta} customClassName={`${prefix}--card__cta`} />
+          <LinkWithIcon href={cta.href}>
+            <span>{cta.copy}</span> <cta.icon.src />
+          </LinkWithIcon>
```

Comment:
I think what we also need to include here is the cta `type` (ie. local, external, jump) that gets lost when we replace with LinkWithIcon. We might need to build that functionality into LinkWithIcon and use the functions from `CTALogic` class (still waiting for that PR to be merged rn)

Reply:
PR is merged in, I think it makes sense to do this approach.

Reply:
@jeffchew , @annawen1 
I've tried to implement the `CTALogic` methods on my latest commit

Reply:
hmm I was expecting the changes to be done on `LinkWithIcon` as well

Reply:
@annawen1 I've updated this PR, now placing the logic on `LinkWithIcon`, thanks!

Reply:
Sorry if the above wasn't too clear, I've made a couple of comments above but I'm wondering if we should keep `LinkWithIcon` as-is, but we can pass in the CTA logic on this level?